### PR TITLE
PS-7081: Null merge PS-5.7.30 into 8.0.20

### PR DESCRIPTION
**Pull request by inikep**
This patch includes `PS-7020: convert MTR tests from python2 to python3 (8.0)` at https://jira.percona.com/browse/PS-7020

### CODE REVIEW

File: mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result [hunk 1]
```diff
@@ -19,3 +20,4 @@ start slave;
 include/stop_slave.inc
 include/start_slave.inc
 include/stop_slave.inc
+include/stop_dump_threads.inc
```
Comment (percona-ysorokin) [inline mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result:23]: From where does this come from?

Comment (inikep) [reply mysql-test/suite/rocksdb_stress/r/rocksdb_stress.result:23]: This test is not tested by Jenkins therefore it was failing before.

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Comment by inikep** [comment]
Tested at https://ps80.cd.percona.com/job/percona-server-8.0-param/645/

**Pull request merged**